en/xml/fo/graphic-pageheader/graphic-pageheader.xsl

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:d="http://docbook.org/ns/docbook">
```

Comment:
You are missing the fo namespace. The root element has to be:

```xsl
<xsl:stylesheet version="1.0"
  xmlns:fo="http://www.w3.org/1999/XSL/Format"
  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
  xmlns:d="http://docbook.org/ns/docbook">
```

Reply:
Funny! I've taken the proposed header from your topic.permalinks.xml.... ;-)
I'll correct this and the previous recommendations.

Reply:
Yes, but the permalinks topic is for HTML. :wink:

Reply:
Done.